src/controllers/plugins/PluginController.cpp

```diff
@@ -306,6 +313,46 @@ const std::map<QString, std::unique_ptr<Plugin>> &PluginController::plugins()
 {
     return this->plugins_;
 }
+std::pair<bool, QStringList> PluginController::updateCustomCompletions(
+    const QString &query, const QString &fullTextContent, int cursorPosition,
+    bool isFirstWord)
+{
+    QStringList results;
+    for (const auto &[name, pl] : getApp()->plugins->plugins())
+    {
+        if (!pl->error().isNull())
+        {
+            continue;
+        }
+        lua::BalanceKepper _(pl->state_);
+        qCDebug(chatterinoLua)
+            << "Processing custom completions from plugin" << name;
```

Comment:
This log entry should be moved into the `if (opt)` part, or it would print for every plugin instead of every plugin that has a completion callback registered